ci-operator/jobs/openshift/cloud-credential-operator/openshift-cloud-credential-operator-master-presubmits.yaml

```diff
@@ -274,6 +274,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-azure,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build03
+    context: ci/prow/e2e-azure-manual-oidc
+    decorate: true
+    labels:
+      ci-operator.openshift.io/cloud: azure4
+      ci-operator.openshift.io/cloud-cluster-profile: azure4
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cloud-credential-operator-master-e2e-azure-manual-oidc
+    optional: true
+    rerun_command: /test e2e-azure-manual-oidc
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
```

Comment:
What are these gcs args and mounts? Do we need them for Azure?

Reply:
These jobs are auto-generated with `make jobs`. Removal of these args will reverted on the next execution of `make jobs` (and I believe there is a ci test will fail while calling it out as such). Therefore, the only thing I modified from the generated jobs is to set `always_run: false`.